addon-test-support/should-force-audit.ts

```diff
@@ -0,0 +1,13 @@
+const _url: URL = new URL(window.location.href, document.baseURI);
+
+export function setEnableA11yAudit(enabled: boolean = false) {
+  if (enabled) {
+    _url.searchParams.set('enableA11yAudit', 'true');
+  } else {
+    _url.searchParams.delete('enableA11yAudit');
+  }
+}
```

Comment:
I think this has conceptually the same issue that I mentioned in the `shouldForceAudit` section. It happens to work because you share the same (stale) `_url` object, but that won't work in general.

```suggestion
export function setEnableA11yAudit(enabled: boolean = false) {
  const url = new URL(window.location.href, document.baseURI);

  // set up the enableA11yAudit query param
  if (enabled) {
    url.searchParams.set('enableA11yAudit', '');
  } else {
    url.searchParams.delete('enableA11yAudit');
  }
  
  // updates the URL without reloading
  window.history.replaceState(null, '', url.href)
}
```